_gsocproposals/2020/proposal_CMS-Monitoring.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: CMS-Monitoring
```

Comment:
here the project name should read "CMS" and not "CMS-Monitoring", otherwise it is not linked with _gsocprojects/2020/project_CMS.md